### PR TITLE
Xenomorph Tweaks

### PR DESCRIPTION
**Pull request by Sun-Soaked**
## About The Pull Request
A number of assorted tweaks to xenomorph functionality that make them better-suited for shiptest

- [x] Removes xenomorph hardstuns, replacing with stamina damage
- [x] Allows aliens to understand, but not speak common
- [x] Allows aliens to pick up and use items. Gives them futzy fingers that prevent firing guns
- [x] Slightly increases xenomorph queen's speed
- [x] Allow aliens to interact with toggleable objects
- [ ] Allow aliens to manipulate computers

## Why It's Good For The Game
-makes xenos both slightly fairer to fight and more of a threat without admin intervention

## Changelog
:cl:
balance: xenomorphs can now pick up items properly. They cannot, however, use firearms.
balance: Replaced all xenomorph hardstuns with stamina damage
balance: xenos can now understand common.
balance: boosts xenomorph queen's speed from Mobility Scooter to John Deere tractor
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
Xenos shouldn't be able to speak common to be honest.

**Comment by MarkSuckerberg** [comment]
> Xenos shouldn't be able to speak common to be honest.

They can't, it just says "understand" common.

**Comment by Sun-Soaked** [comment]
they can't, at least not by default- they can only understand it.

> Xenos shouldn't be able to speak common to be honest.

**Comment by ValoTheValo** [comment]
And being able to do stuff like weld doors or fly a fucking space ship doesnt make sense. They are smart but not that smart and also they dont have thumbs

**Comment by Sun-Soaked** [comment]
> And being able to do stuff like weld doors or fly a fucking space ship doesnt make sense. They are smart but not that smart and also they dont have thumbs

fun fact: aliens actually have mirrored opposable thumbs on each hand.
image

**Comment by TallDeekus** [comment]
Honestly based PR, although I wonder if there is a way to make it so that, while they can use consoles and stuff, the typing on the console is randomized, understanding spoken common does not equal being able to read it, it would be neat, as it wouldn't limit players controlling computers, but it would limit information retained from them.

**Comment by Dimasw99** [comment]
Give them DNA too while you are at it so that they can be cloned

**Comment by UltraNumeron** [comment]
Does this mean benos could now hypothetically use a holopara injector

**Comment by ValoTheValo** [comment]
> Does this mean benos could now hypothetically use a holopara injector

It means benos can robust people with toolboxes and eswords and yes holopara

**Comment by UltraNumeron** [comment]
that is hilarious and i love it

**Comment by ValoTheValo** [comment]
My favorite scene in alien is when the xenomorph hit lampert over the head with a toolbox 12 timed and said "skill issue cope and seethe" then hacked the atmospherics and vented the entire nostromo by setting it to panic siphon

**Comment by ilikesaying** [comment]
My favourite scene in Aliens was when the Queen used her stand "「Danger Zone」" to trap the Colonial Marines inside the room while the drones throw paper airplanes and soap.

**Comment by Sun-Soaked** [comment]
I need some help with task 6/6, if anyone familiar with buttons in TGUI tabs ends up reading this

**Comment by ghost** [comment]
Good lord i can imagine a xeno with a fireaxe or a double esword.

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -277,16 +277,16 @@
...
 
-		if(M.a_intent == INTENT_DISARM) //Always drop item in hand, if no item, get stun instead.
+		if(M.a_intent == INTENT_DISARM) //heavy stamina, 50% chance to knock item out of hand
 			var/obj/item/I = get_active_held_item()
-			if(I && dropItemToGround(I))
+			if(I && dropItemToGround(I) & prob(50))
 				playsound(loc, 'sound/weapons/slash.ogg', 25, TRUE, -1)
 				visible_message("<span class='danger'>[M] disarms [src]!</span>", \
 								"<span class='userdanger'>[M] disarms you!</span>", "<span class='hear'>You hear aggressive shuffling!</span>", null, M)
 				to_chat(M, "<span class='danger'>You disarm [src]!</span>")
```
Comment (MarkSuckerberg) [inline code/modules/mob/living/carbon/human/human_defense.dm]:
```suggestion
			if(I && dropItemToGround(I))
				if(prob(50))
					playsound(loc, 'sound/weapons/slash.ogg', 25, TRUE, -1)
					visible_message("<span class='danger'>[M] disarms [src]!</span>", \
								"<span class='userdanger'>[M] disarms you!</span>", "<span class='hear'>You hear aggressive shuffling!</span>", null, M)
					to_chat(M, "<span class='danger'>You disarm [src]!</span>")
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by MarkSuckerberg** [comment]
This is funny, and the code most definitely works, but there are severe balance concerns that would require a great deal of effort to reconcile, I believe.

**Comment by MarkSuckerberg** [comment]
General maintainer consensus that this is way too powerful. I'm sure there's different avenues that could be explored in the same vein as this, such as a xeno race or something similar, but as it stands this just won't end well.

**Pull request closed without merging**